client/src/components/EmailAlertSignup.tsx

```diff
@@ -20,6 +20,7 @@ import { AddressRecord } from "./APIDataTypes";
 import SendNewLink from "./SendNewLink";
 
 const SUBSCRIPTION_LIMIT = 15;
+const BRANCH_NAME = process.env.REACT_APP_BRANCH;
```

Comment:
this feels so redundant to keep declaring const at the top but to make a const global in Typescript didn't really seem like a sound option. lmk if you have any thoughts to make this look better!

Reply:
hmm yeah I can't think of anything better right now. Seems fine for now, especially if we're going to try for a global solution later so it doesn't need to be passed with each event.